apps/omg/lib/omg/ethereum_event_listener/groomer.ex

```diff
@@ -0,0 +1,31 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.EthereumEventListener.Groomer do
```

Comment:
`Preprocessor` maybe?

Reply:
ah, that was my first take actually, but then had a thought we're overloading the word "process" too much. I'll revert to this then.